src/pages/categoryOptionGroupSets/List.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import { DefaultSectionList } from '../DefaultSectionList'
+
+export const Component = () => (
+    <DefaultSectionList filters={['name:ne:default']} />
```

Comment:
We don't actually need `default` filter here. There's no default for groups or group sets.